src/app/commands/commands.component.ts

```diff
@@ -35,7 +35,8 @@ export class CommandsComponent implements OnInit {
   scrollTo(id: string) {
     const el = document.getElementById(id);
     if (el) {
-      el.scrollIntoView({behavior: 'smooth', block: 'center'});
+      // do this *after* any dialog closing/animations
+      window.setTimeout(() => el.scrollIntoView({behavior: 'smooth', block: 'center'}), 0);
```

Comment:
In most cases when using `0` for the timeout delay, what you really want is `requestAnimationFrame`.

Functionally, this is very similar to `setTimeout(..., 0)`, but it gives the browser a bit more control in when specifically it executes the callback. Especially if the callback does anything that could affect layout/paint/compositing steps (position, color, content, scrolling, etc), `requestAnimationFrame` will help the app stay at 60fps where `setTimeout(..., 0)` could cause it to skip a frame now and then, which in aggregate, could become noticible to the user.

https://developer.mozilla.org/en-US/docs/Web/API/window/requestAnimationFrame

https://medium.com/javascript-in-plain-english/better-understanding-of-timers-in-javascript-settimeout-vs-requestanimationframe-bf7f99b9ff9b